<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLayoutDialog</class>
 <widget class="QDialog" name="AddLayoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Layout</string>
  </property>
  <layout class="QVBoxLayout" name="vBoxLayout">
   <item row="0">
    <widget class="QLineEdit" name="layoutSearchField">
     <property name="placeholderText">
      <string>Searchâ€¦</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1">
    <widget class="QListWidget" name="layoutListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="shortcutLabel">
       <property name="text">
        <string>Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KKeySequenceWidget" name="kkeysequencewidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLabel">
       <property name="text">
        <string>Label:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="labelEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3">
    <layout class="QHBoxLayout">
     <item column="0">
      <widget class="QPushButton" name="prevbutton">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>labelEdit</tabstop>
  <tabstop>prevbutton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLayoutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLayoutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
